samples/cpp/speech_sample/README.md

```diff
@@ -228,9 +228,9 @@ Kaldi's nnet-forward command. Since the `speech_sample` does not yet use pipes,
    cat out.txt | utils/int2sym.pl -f 2- words.txt | sed s:\<UNK\>::g | compute-wer --text --mode=present ark:test_filt.txt ark,p:-
    ```
 
-   All of mentioned files can be downloaded from [https://storage.openvinotoolkit.org/models_contrib/speech/2021.2/wsj_dnn5b_smbr](https://storage.openvinotoolkit.org/models_contrib/speech/2021.2/wsj_dnn5b_smbr)
+   All of mentioned files can be downloaded from the [storage](https://storage.openvinotoolkit.org/models_contrib/speech/2021.2/wsj_dnn5b_smbr)
```

Comment:
All of the mentioned files can be downloaded from [storage]

Reply:
Done in https://github.com/openvinotoolkit/openvino/pull/14197/commits/8397872e5f1da428cc20134d62925dfc851e40c0